src/transformers/models/vision_encoder_decoder/configuration_vision_encoder_decoder.py

```diff
@@ -119,3 +127,85 @@ def to_dict(self):
         output["decoder"] = self.decoder.to_dict()
         output["model_type"] = self.__class__.model_type
         return output
+
+
+class VisionEncoderDecoderEncoderOnnxConfig(OnnxConfig):
+    torch_onnx_minimum_version = version.parse("1.11")
+
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        return OrderedDict(
+            [
+                ("pixel_values", {0: "batch", 1: "num_channels", 2: "height", 3: "width"}),
+            ]
+        )
+
+    @property
+    def atol_for_validation(self) -> float:
+        return 1e-4
+
+    @property
+    def outputs(self) -> Mapping[str, Mapping[int, str]]:
+        return OrderedDict({"last_hidden_state": {0: "batch", 1: "encoder_sequence"}})
```

Comment:
nit: it could be helpful to indicate which hidden state we're referring to explicitly

```suggestion
        return OrderedDict({"encoder_last_hidden_state": {0: "batch", 1: "encoder_sequence"}})
```

Reply:
There is a output name check between the reference model and ONNX exported model. Thus if we change the name it results in an error: `Outputs doesn't match between reference model and ONNX exported model: {'encoder_last_hidden_state}'`